File: 151001/Aleksandrov/RESTLab1/src/main/java/by/bsuir/dto/CommentRequestTo.java
package by.bsuir.dto;

import jakarta.validation.constraints.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.format.annotation.NumberFormat;

import java.util.Set;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CommentRequestTo {
    private Long id;
    @NotBlank
    @Size(min = 2, max = 2048)
    private String content;
    private Long issueId;
}

File: 151004/Gonzarevich/lab1/DTO/CommentRequestTo.php
<?php

namespace App\DTO;

use Symfony\Component\Validator\Constraints as Assert;

class CommentRequestTo
{
    public function __construct(

        public readonly ?int $id,

        public readonly ?int $issueId,

        #[Assert\NotBlank]
        #[Assert\Length(min: 4, max: 2048)]
        public readonly string $content
    ) {
    }
}